mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -451,4 +459,62 @@ private String buildOrderByClause(List<SortField> sortFields) {
       .collect(Collectors.joining(", ", "ORDER BY ", EMPTY));
   }
 
+  @Override
+  public boolean permanentDeleteJobExecutions(String tenantName) {
+    Promise<RowSet<Row>> promise = Promise.promise();
+    Promise<SQLConnection> connection = Promise.promise();
+    String rollbackMessage = "Rollback transaction. Error during jobExecution deletion process.";
+    Future.succeededFuture()
+      .compose(c -> {
+        pgClientFactory.createInstance(tenantName).startTx(connection);
```

Comment:
We can move statement: pgClientFactory.createInstance(tenantName) to upper level and not create it each time something like this:
PostgresClient pgClient = pgClientFactory.createInstance(tenantId);

Reply:
Hi @SerhiiNosko - Updated code as per your suggestion. Thank you 🙂